src/templates/best-cards/clans/ahrimanes.html

```diff
@@ -1,10 +1,10 @@
 {% extends "best-cards/_layout.html" %}
-{% import "best-cards/_layout_card_column.html" as layout with context%}
+{% import "best-cards/_layout_card_column.html" as layout with context %}
 
 {% block content %}
 <h1><img src="{{ url_for('static', filename='img/icons/clan-ahrimanes.svg') }}"> Ahrimanes</h1>
 <div class="tip">
-    {% trans trimmed 
```

Comment:
Atom.io is automatically removing needless eol blanks for examples

Reply:
I don't really mind - I may just have different settings on my VisualCode instance, but that should not matter much